packages/cli/src/api/extractors/typescript.ts

```diff
@@ -1,63 +1,13 @@
-import fs from "fs"
-import { transform } from "@babel/core"
-import linguiExtractMessages, {ExtractPluginOpts} from "@lingui/babel-plugin-extract-messages"
-
-import { projectType } from "../detect"
-import { ExtractorType, BabelOptions } from "."
-
-const typescriptRe = /(^.?|\.[^d]|[^.]d|[^.][^d])\.tsx?$/i
+import { ExtractorType } from "."
 
 const extractor: ExtractorType = {
   match(filename) {
-    return typescriptRe.test(filename)
+    throw new Error("Typescript extractor was removed. " +
+      "Lingui CLI can parse typescript out of the box. " +
+      "Please remove it from your lingui.config.js");
```

Comment:
That was deprecated long time ago, now it's completely unused. People still used it because of burden in resulting babel configuration, so for some configurations this worked better.  Now it's completely unneeded.